package.json

```diff
@@ -47,18 +47,24 @@
     "benchmark": "^2.0.0",
     "chai": "^3.0.0",
     "coveralls": "^2.11.2",
+    "electron": "1.6.11",
+    "electron-mocha": "4.0.0",
     "istanbul": "^0.4.4",
     "mocha": "^3.2.0",
+    "nyc": "11.0.3",
     "sinon": "^1.14.1"
   },
   "scripts": {
-    "prepublish": "npm run build && mocha",
-    "test": "npm run build && mocha && istanbul report",
-    "coverage-report": "istanbul report text-summary lcov",
+    "prepublish": "npm run test",
+    "test": "npm run build && nyc mocha --growl",
+    "test:electron": "npm run build && nyc electron-mocha --renderer test/**/*.coffee",
```

Comment:
mocha & electron-mocha both use `mocha.opts` file.
I added `test:electron` command to run the full test suite in electron context, which was a simple and good-enough test

But because electron-mocha doesn't support `--growl` flag, I had to move it above.

I although used nyc instead of the manual istanbul invocation to produce report.